analyses/infercnv-consensus-cell-type/scripts/build-normal-reference/build-reference-SCPCP000015.R

```diff
@@ -114,6 +114,9 @@ clean_sce <- function(sce) {
   reducedDim(sce, "PCA") <- NULL
   reducedDim(sce, "UMAP") <- NULL
 
+  # ensure the counts matrix is sparse
+  counts(sce) <- as(counts(sce), "dgCMatrix")
```

Comment:
Are you using the merged objects? Maybe we should re-process the merged objects in the workflow results bucket so that they match the formatting of what's available on the Portal which should now be sparse.